frontend/src/components/MenuItem.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Image from "next/image";
+interface MenuItemProps {
+    name: string;
+    price: number;
+    picture: string;
+    addedToOrder: boolean;
+    addItem: (name: string, price: number) => void;
+}
+export default function MenuItem({ name, price, picture, addItem }: MenuItemProps) {
+  const imageSize = 80;
+  return (
+    <div className="menu-item">
+      <div className="menu-item-picture-and-price-group">
+        <div className="menu-item-picture">
+          <Image src={picture} width={imageSize} height={imageSize} alt={name}/>
+        </div>
+        <div className="menu-pricing-group">
+          <div>Price:</div>
```

Comment:
We should use spans here. Or a semantic tag if possible.

Reply:
fixed with spans, and better styling to make sure the buttons align regardless of the text's length.